frontend/index.html

```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>transcribee 🐝</title>
-    <link rel="stylesheet" href="src/index.css">
-    <link rel="stylesheet" href="node_modules/@fontsource/inter/latin.css">
+    <link rel="stylesheet" href="/src/index.css">
+    <link rel="stylesheet" href="/node_modules/@fontsource/inter/latin.css">
```

Comment:
Can we make this (and the routes) depend on a base url setting? So we can host transcribee somewhere else than the document root?

Reply:
Done, the router now respects vite's base path (https://vitejs.dev/guide/build.html#public-base-path).